### DISCUSSION TITLE
Go back to 'Normal Drawing'

### DISCUSSION THREAD

**Discussion by pnclvo** [xournalpp/xournalpp#2318] [Q&A] [answered]
Hi,
when I draw an object (a circle, a box..) I cannot find a fast way to go back to 'normal drawing' (without stroke recognition).
I usually click 'Ctrl+1' two times, but maybe a more direct way exists... if so, please tell me!
Using 'Default Tool' is another possibility but resets also color and pen size
I would suggest adding an entry 'Normal Drawing' in the 'Shape' list, with an associated shortcut

Thanks,
pnc

**Comment by Technius** [comment, marked as answer]
Hi pnclvo, you can just click the selected shape type to toggle it:

![xpp_2318](https://user-images.githubusercontent.com/1066652/96170287-421f3b80-0ed8-11eb-8deb-7adf9a39327d.gif)

If you want to make it more convenient, you may want to check out the [experimental floating toolbar feature](https://github.com/xournalpp/xournalpp/issues/804#issuecomment-498475992). To use it, assign `Floating Toolbar` to one of your button actions in the settings, and then use `View -> Toolbars -> Customize` to drag and drop icons on to the toolbar.

**Comment by pnclvo** [comment]
Hi Bryan, you gave me two very useful hints.
Thank you so much!
